src/sparse_ir/kernel.py

```diff
@@ -85,6 +89,10 @@ def conv_radius(self):
         """
         return None
 
+    def weight_func(self, statistics: str):
+        """ Return the weight function for given statistics """
+        pass
```

Comment:
I find this really useful!

But I think we should provide a sensible default here for implementers:
```suggestion
    def weight_func(self, statistics: str):
        """Return the weight function for given statistics"""
        if statistics not in 'FB':
            raise ValueError("statistics must be 'F' for fermions or 'B' for bosons")
        return lambda x: np.ones_like(x)
```